portfolio/src/main/webapp/script.js

```diff
@@ -60,9 +60,10 @@ function fetchBlobstoreUrlAndShowForm() {
       })
       .then((imageUploadUrl) => {
         const messageForm = document.getElementById('my-form');
+        console.log(imageUploadUrl);
         messageForm.action = imageUploadUrl;
         messageForm.classList.remove('hidden');
-      });
+    });
```

Comment:
Nit: The previous indentation seemed right. This line should be indented the same as the ".then" before it.